commands/system-prune.ts

```diff
@@ -11,6 +12,8 @@ import { ext } from '../extensionVariables';
 import { docker } from './utils/docker-endpoint';
 
 export async function systemPrune(actionContext: IActionContext): Promise<void> {
```

Comment:
I prefer `registerCommand('vscode-docker.system.prune', systemPrune);` to keep dockerExtension.ts as simple as possible.

Which means you would only change this file:
```typescript
export async function systemPrune(this: IActionContext): Promise<void> {
    const actionContext = this; // <-- if you don't like referencing `this` directly
```

But no strong preference.

Reply:
I like the idea but don't want to deal with changing it now, but will keep it in mind in the future. Thx.